src/likes/likes.service.ts

```diff
@@ -51,7 +56,8 @@ export class LikesService {
                 id
             }
         });
-        return deletedLike
+        const action = await this.actionsService.addAction('Delete',userId,'PostLike',likePost.id,deletedLike)
```

Comment:
here you have hadcoded strings. Store them in variables and reuse? insted of hardcode them